docs/sec_installation.rst

```diff
@@ -9,11 +9,35 @@ including:
  - `scipy <https://docs.scipy.org/doc/scipy/reference/>`_, and
  - `scikit-image <http://scikit-image.org/>`_ (phase unwrapping using :py:func:`skimage.restoration.unwrap_phase`).
     
+Base Package
+------------
 
 To install qpretrieve, use one of the following methods
-(package dependencies will be installed automatically):
+(mandatory package dependencies will be installed automatically):
     
 * from `PyPI <https://pypi.python.org/pypi/qpretrieve>`_:
     ``pip install qpretrieve``
 * from `sources <https://github.com/RI-imaging/qpretrieve>`_:
-    ``pip install -e .`` or
+    ``pip install -e .``
+
+Optional Dependencies
+---------------------
+
+``qpretrieve`` has two optional dependencies:
+`PyFFTW <https://pyfftw.readthedocs.io/en/latest/>`_ and
+`CuPy <https://cupy.dev/>`_.
+
+They can be installed individually or all at once:
+
+   ``pip install qpretrieve[FFTW]``
+
+   ``pip install qpretrieve[FFTW, CUPY]``
```

Comment:
I think the space after the comma will break this command (at least on older versions of pip).